airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/AcceptanceTests.java

```diff
@@ -166,19 +174,26 @@
 
   @BeforeAll
   public static void init() {
-    sourcePsql = new PostgreSQLContainer("postgres:13-alpine")
-        .withUsername(SOURCE_USERNAME)
-        .withPassword(SOURCE_PASSWORD);
-    sourcePsql.start();
+    if (!IS_GKE) {
```

Comment:
can you update the class doc comment to say that the source/dest db are actual pods when testing this on GKE?